kobo/settings/base.py

```diff
@@ -640,10 +640,14 @@ def __init__(self, *args, **kwargs):
     AWS_SES_REGION_ENDPOINT = os.environ.get('AWS_SES_REGION_ENDPOINT')
 
 
+
 ''' Storage configuration '''
 if 'KPI_DEFAULT_FILE_STORAGE' in os.environ:
-    # To use S3 storage, set this to `storages.backends.s3boto3.S3Boto3Storage`
+    # To use S3 storage, set this to `kobo.apps.storage_backends.s3boto3.S3Boto3Storage`
     DEFAULT_FILE_STORAGE = os.environ.get('KPI_DEFAULT_FILE_STORAGE')
+    if DEFAULT_FILE_STORAGE == 'storages.backends.s3boto3.S3Boto3Storage':
+        # Force usage of custom S3 tellable Storage
+        DEFAULT_FILE_STORAGE = 'kobo.apps.storage_backends.s3boto3.S3Boto3Storage'
```

Comment:
👏   

I was going to say: we need to open an issue in kobo-install to support the new class. 

- https://github.com/kobotoolbox/kobo-install/blob/5d8cd0c0c9fdd265d0be9cdcdc1ba47fd435585e/templates/kobo-env/envfiles/aws.txt.tpl#L14
- https://github.com/kobotoolbox/kobo-install/blob/5d8cd0c0c9fdd265d0be9cdcdc1ba47fd435585e/templates/kobo-env/envfiles/aws.txt.tpl#L17

Reply:
Yeah I'd rather no one know we do this horrible hack :smile: and we may some day be able to remove it. So we don't want under users changing it.